src/wp-includes/js/media/controllers/gallery-add.js

```diff
@@ -71,15 +82,19 @@ GalleryAdd = Library.extend(/** @lends wp.media.controller.GalleryAdd.prototype
 			library.unobserve( this.editLibrary );
 		}
 
-		// Accepts attachments that exist in the original library and
-		// that do not exist in gallery's library.
+		/*
+		 * Accept attachments that exist in the original library and
```

Comment:
`Accept attachments that exist in the original library but not exist in gallery's library yet`?